src/django_scim/views.py

```diff
@@ -46,7 +81,7 @@ def login_required(
 
     This function was pulled almost directly from Django's implementation.
     """
-    actual_decorator = user_passes_test(
+    actual_decorator = request_passes_test(
         get_is_authenticated_predicate(),  # <- Only line that differs from Django's implementation.
```

Comment:
No longer the only line that differs from Django’s implementation.

Reply:
Yep.